fastlane/Fastfile

```diff
@@ -92,6 +96,10 @@ end
 
 desc "Completes an SDK Release"
 lane :publish_release do |options|
+
+  # Make xcframeworks
+  make frameworks
```

Comment:
this will not upload the frameworks, those should be attached to the github release

Reply:
Do we want to attach all 3 or just StreamChat & StreamChatUI xcframework?

Reply:
All 3 are needed (Carthage uses `StreamChat-All`, and SPM and CocoaPods use the others )